Examples/README.md

```diff
@@ -1,12 +1,12 @@
-ITK Examples Directory
+ITK Sphinx Examples Directory
 ----------------------
```

Comment:
Add more ---

Should we switch to `# Title` notation?

Reply:
:+1: Updated